test/react/hooks/use-occupancy.test.tsx

```diff
@@ -37,7 +37,7 @@ describe('useOccupancy', () => {
     // create a new mock room before each test, enabling occupancy
     vi.resetAllMocks();
     mockLogger = makeTestLogger();
-    mockRoom = makeRandomRoom({ options: { occupancy: true } });
+    mockRoom = makeRandomRoom({ options: { occupancy: {} } });
```

Comment:
Why have we changed this for a dep update? 👀

Reply:
The types in the test were wrong - it's supposed to be the empty object.

I tightened up the definition in the main `RoomOptions` to match what it should be (it should be an object, but defining it as an `interface` made it allow booleans too) which then meant I needed to update tests.

Reply:
Oh I see, the interface changed